gulpfile.js

```diff
@@ -854,55 +854,51 @@ gulp.task('package-json', function(cb) {
   fs.writeFileSync(`${packageDistribution}/package.json`,
       JSON.stringify(json, null, 2));
   cb();
-});
+};
 
 /**
  * This task copies the package/README.md file into the distribution directory.
  * This file is what developers will see at https://www.npmjs.com/package/blockly.
  */
-gulp.task('package-readme', function() {
+function packageReadme() {
   return gulp.src('./package/README.md')
-    .pipe(gulp.dest(`${packageDistribution}`))
-});
+    .pipe(gulp.dest(`${packageDistribution}`));
+};
 
 /**
  * This task copies the typings/blockly.d.ts TypeScript definition file into the
  * distribution directory.
  * The bundled declaration file is referenced in package.json in the types property.
  */
-gulp.task('package-dts', function() {
+function packageDTS() {
   return gulp.src('./typings/blockly.d.ts')
-    .pipe(gulp.dest(`${packageDistribution}`))
-});
+    .pipe(gulp.dest(`${packageDistribution}`));
+};
 
 /**
  * This task prepares the NPM distribution files under the /dist directory.
  */
-gulp.task('package', gulp.parallel(
-  'package-index',
-  'package-browser',
-  'package-node',
-  'package-core',
-  'package-node-core',
-  'package-blockly',
-  'package-blockly-node',
-  'package-blocks',
-  'package-javascript',
-  'package-python',
-  'package-lua',
-  'package-dart',
-  'package-php',
-  'package-locales',
-  'package-media',
-  'package-umd-bundle',
-  'package-json',
-  'package-readme',
-  'package-dts'
-  ));
-
-// The default task builds Blockly.
-gulp.task('default', gulp.series(['build']));
```

Comment:
What happens to default? ie: when someone runs ``gulp``.

Reply:
I'm exporting default: build at the bottom.